src/main/java/org/jenkinsci/plugins/workflow/cps/Snippetizer.java

```diff
@@ -535,7 +535,7 @@ public HttpResponse doGenerateSnippet(StaplerRequest req, @QueryParameter String
         return ExtensionList.lookup(SnippetizerLink.class);
     }
 
-    @Restricted(DoNotUse.class)
+    @Restricted(NoExternalUse.class)
```

Comment:
Seems unrelated?

Reply:
Mandated by updating the parent POM; works around https://github.com/kohsuke/access-modifier/pull/12.